tests/ChangeLog

```diff
@@ -761,7 +767,7 @@
 	* testsuite.src/*.at: Added check for cobc's exit code and stderr
 		where missing
 
-2014-14-04  Philipp B�hme <phi.boehme@googlemail.com>
+2014-14-04  Philipp B�hme <phi.boehme@googlemail.com>
```

Comment:
somethings broken here, maybe you can revert the line (otherwise I can do it after merging)

Reply:
I'll take care of that in the next commit